src/components/TopologySidebar/TopologyCost.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from "react";
+import { Topology } from "../../context/TopologyPageContext";
+import {
+  arrangeCostDataForInfoPanel,
+  CostInfoPanel,
+  isCostsDataEmpty
```

Comment:
Why do we need the arrangeCost / isCosts empty functions - can't those be called from inside the CostInfoPanel

Reply:
These are exposed because of following reasons.
- we don't want to hard code the view behaviour into the CostInfoPanel in case of data absence.
- cost data format can be different so we want to covert that into the format ConstInfoPanel understands.
   - we provided one implementation for our existing cost data format.
   
I don't have strong opinions around this. If we get the cost data in same format across all api's and we don't have use case for having different ui behaviour in case of cost data absence. I can move this into CostInfoPanel component.

@moshloop please let me know your thoughts.

Reply:
> we don't want to hard code the view behaviour into the CostInfoPanel i 

Why not, if the data is empty it doesn't render anything, or renders an empty state if specified

> cost data format can be different

It shouldn't be